src/components/UserBubble/UserBubble.vue

```diff
@@ -90,6 +90,7 @@ This component has the following slot:
 				:style="styles.avatar"
 				:disable-tooltip="true"
 				:disable-menu="true"
+				:show-user-status="showUserStatusOnAvatar"
```

Comment:
should maybe just `v-bind="$props"` so everything is forwarded to the Avatar?

Reply:
Fine by me, if we have no conflicting things?